cli/src/factory_ops/unboxing/init

```diff
@@ -336,33 +336,12 @@ echo "Removing the repos with Seagate URL from Server A" | tee -a ${LOG_FILE}
 seagate_refs_cleanup
 
 echo "Restarting the cluster resources.........." | tee -a ${LOG_FILE}
```

Comment:
Change this sentence preferably mentioning about cluster restart (through switching maintenance mode)